src/Bahyph.xml

```diff
@@ -0,0 +1,29 @@
+<SPDX name="Bahyph License" identifier="Bahyph" osi-approved="false">
+  <urls>
+    <url>https://fedoraproject.org/wiki/Licensing/Bahyph</url>
+  </urls>
+  <license>
+    <optional>
+      <p>COPYRIGHT NOTICE</p>
+    </optional>
+    <body>
+      <p>These patterns and the generating sh script are Copyright (c) GMV 1991</p>
```

Comment:
discussed on call; decided not to add copyright tag and never had matching markup, so not adding it now as more of a one-off license